### DISCUSSION TITLE
Node URI does not load port from settings

### DISCUSSION THREAD

**Discussion by mbio16** [Ride-The-Lightning/RTL#791] [Q&A] [answered]
<img width="1010" alt="Snímek obrazovky 2021-09-06 v 15 37 54" src="https://user-images.githubusercontent.com/43006943/132228784-ee753fe1-1a87-459c-951d-e822e6082d09.png">

<img width="1010" alt="Snímek obrazovky 2021-09-06 v 15 55 10" src="https://user-images.githubusercontent.com/43006943/132228794-77690031-c6b3-425d-a237-bdb86dbf1640.png">

- seems that port is hardcoded
#### context
- LND node in regtest lab

**Comment by saubyk** [comment, marked as answer]
Hi @mbio16 
The port is not hardcoded. RTL is displaying the values as configured in your `lnd.conf` file.
The reason your port is not showing up per your configuration is, because LND does not set the port value from `listen` param for external access. Instead the value for the port is picked up from `externalip` param. You should configure your externalip as `x.x.x.x:9734` instead.

**Reply by mbio16** [reply]
thanks for reply.
